widgets/src/components/TaskDetails/Overview/OverviewHeader.js

```diff
@@ -1,21 +1,34 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Typography from '@material-ui/core/Typography';
 import Box from '@material-ui/core/Box';
+import Link from '@material-ui/core/Link';
+import Typography from '@material-ui/core/Typography';
 
-import OverviewNavigation from 'components/TaskDetails/Overview/OverviewNavigation';
+// import OverviewNavigation from 'components/TaskDetails/Overview/OverviewNavigation';
```

Comment:
will be needed?

Reply:
it will be reworked in the future, so I didn't removed completely